src/engagement_db_to_analysis/engagement_db_to_analysis.py

```diff
@@ -97,37 +97,72 @@ def _convert_messages_to_traced_data(user, messages_map):
     """
     Converts messages dict objects to TracedData objects.
 
-    :param user:
+    :param user: Identifier of user running the pipeline.
     :type user: str
     :param messages_map: Dict containing messages data.
     :type messages_map: dict
     :return: A list of Traced data message objects.
     :type: list of Traced data
     """
 
-    messages_td = []
+    messages_traced_data = []
     for engagement_db_dataset in messages_map:
 
         engagement_db_dataset_messages = messages_map[engagement_db_dataset]
         for msg in engagement_db_dataset_messages:
-            messages_td.append(
+            messages_traced_data.append(
                 TracedData(msg, Metadata(user, Metadata.get_call_location(), TimeUtils.utc_now_as_iso_string())))
 
-    log.info(f"Converted {len(messages_td)} raw messages to TracedData")
+    log.info(f"Converted {len(messages_traced_data)} raw messages to TracedData")
 
-    return messages_td
+    return messages_traced_data
 
-#TODO: Fold messages by uid
-def _fold_messages_by_uid():
-    return None
+def _fold_messages_by_uid(user, messages_traced_data):
+    """
+    Groups Messages TracedData objects into Individual TracedData objects.
+
+    :param user: Identifier of user running the pipeline.
+    :type user: str
+    :param messages_traced_data: Messages TracedData objects to group.
+    :type messages_traced_data: list of TracedData
+    :return: Individual TracedData objects.
+    :rtype: dict of uid -> individual TracedData objects.
+    """
+
+    participants_traced_data = {}
```

Comment:
```suggestion
    participants_traced_data_map = {}
```
since it's not traced data

Reply:
Updated thanks.